src/site/styles/layout.less

```diff
@@ -99,15 +134,10 @@ input {
     }
 }
 
-#logout .action-x {
+#logout .action-Sign-Out {
```

Comment:
Instead of having this ugly class name, how about lowercasing the class name in the component?